live-hls.js

```diff
@@ -91,7 +97,11 @@ getHeaderObjects = function(fileContent) {
       header.MediaSequence.value = lines[i].substr(indexOfColon + 1,lines[i].length-indexOfColon);
       continue;
     }
-    //Todo: Add Discontinuity Sequence
+    if (lines[i].indexOf('#EXT-X-DISCONTINUITY-SEQUENCE:') === 0) {
+      header.Discontinuity.tag = '#EXT-X-DISCONTINUITY-SEQUENCE';
```

Comment:
Previously, I add a discontinuity to the segment, but Discontinuity-sequence is new as it tracks all dropped discontinuities.